arex-schedule-web-api/src/main/java/com/arextest/schedule/planexecution/impl/PlanExecutionMonitorImpl.java

```diff
@@ -34,52 +32,56 @@ public void monitorOne(ReplayPlan task) {
     for (PlanMonitorHandler handler : planMonitorHandlerList) {
       try {
         handler.handle(task);
-      } catch (Throwable t) {
-        LOGGER.error("Error handling plan:{}", task.getId(), t);
+      } catch (Exception e) {
+        LOGGER.error("Error handling plan:{}", task.getId(), e);
       }
     }
   }
 
   @Override
   public void register(ReplayPlan plan) {
```

Comment:
What if I have duplicate registrations?

Reply:
The monitorScheduler is injected through the @Resource annotation and is a singleton. Actually more tasks are scheduled on the same ScheduledExecutorService instance (i.e. monitorScheduler), which means these tasks will share the same thread pool.

Reply:
If a replayPlan is re-registered multiple times, duplicate MonitorTasks are created